### PR TITLE
ci: show version in controller

### PR DESCRIPTION
**Pull request by anda-ren**
## Description
the version of controller will show like `0.3.0-rc.5-nightly-20220920-a63f51c7:0.1.0.dev15:https://github.com/anda-ren/starwhale/tree/show_version`

`0.3.0-rc.5-nightly-20220920-a63f51c7` means the version of controller
`0.1.0.dev15` means the version of swcli
`https://github.com/anda-ren/starwhale/tree/show_version` means the path of the repo
## Modules
- [ ] UI
- [ ] Controller
- [ ] Agent
- [ ] Client
- [ ] Python-SDK
- [x] Others

## Checklist
- [ ] run code format and lint check
- [ ] add unit test
- [ ] add necessary doc

### CODE REVIEW

File: docker/Dockerfile.server [hunk 1]
```diff
@@ -1,12 +1,16 @@
 ARG BASE_SERVER_IMAGE=starwhaleai/base_server:latest
 FROM ${BASE_SERVER_IMAGE}
+ARG SW_SERVER_VERSION=server_version_not_set
+ARG SW_VERSION=cli_version_not_set
```
Comment (tianweidut) [inline docker/Dockerfile.server:4]: remove `SW_VERSION`?  it is the same as `SW_VERSION`.

Comment (anda-ren) [reply docker/Dockerfile.server:4]: the version of controller will show like `0.3.0-rc.5-nightly-20220920-a63f51c7:0.1.0.dev15:https://github.com/anda-ren/starwhale/tree/show_version`

`0.3.0-rc.5-nightly-20220920-a63f51c7` means the version of controller
`0.1.0.dev15` means the version of swcli(pypi)

Comment (anda-ren) [reply docker/Dockerfile.server]: done

**Review by tianweidut — APPROVED** [review]
LGTM

**Review by tianweidut — APPROVED** [review]

**Pull request merged**